components/vf-card/vf-card.jsx

```diff
@@ -0,0 +1,75 @@
+import { VfClassNormalize } from "../../tools/vf-extensions-react/vf-extensions-react";
```

Comment:
I think once shipped to npm this should be:

`import { VfClassNormalize } from "@visual-framework/vf-extensions-react/vf-extensions-react";`